.github/workflows/run-lib-injection.yml

```diff
@@ -7,55 +7,65 @@ on:
         description: "Shall we build python base images for tests on python tracer"
         default: false
         required: false
-        type: boolean 
+        type: boolean
+      library:
+        description: "Library to test"
+        required: true
+        type: string
 env:
   REGISTRY: ghcr.io
 
 jobs:
+
+  compute-matrix:
+    runs-on: ubuntu-latest
+    outputs:
+      matrix: ${{ steps.compute-matrix.outputs.matrix }}
+    steps:
+    - name: Compute matrix
+      id: compute-matrix
+      shell: python
+      run: |
+        import json
+        import os
+
+        weblogs={
+          "cpp": [],
+          "dotnet": ["dd-lib-dotnet-init-test-app"],
+          "golang": [],
+          "java": ["dd-lib-java-init-test-app"],
+          "nodejs": ["sample-app"],
+          "php": [],
+          "python": ["dd-lib-python-init-test-django", "dd-lib-python-init-test-django-gunicorn", "dd-lib-python-init-test-django-uvicorn"],
+          "ruby": ["dd-lib-ruby-init-test-rails", "dd-lib-ruby-init-test-rails-explicit", "dd-lib-ruby-init-test-rails-gemsrb"]
+        }
+
+        result = weblogs["${{ inputs.library }}"]
```

Comment:
I guess we always going to launch the lib-injection tests for all the libraries, because the weblog are located on the "/lib-injection/docker/lang folder".
But for me is OK. The idea is move the lib-injection weblogs to utils/build/docker

Reply:
ah, I can add it, it's quite easy

Reply:
done [44de5d8](https://github.com/DataDog/system-tests/pull/2356/commits/44de5d82e710548cfdf91056ef47b2578c504743)

Don't forget to remove it when you'll merge docker folders :)